examples/requirements.txt

```diff
@@ -1,4 +1,6 @@
-promptflow[azure]
-promptflow-tools==0.1.0.b5
+promptflow[azure]==0.0.111997232
```

Comment:
need to change the PR title. shall this just be the release PR?

Reply:
No, this is just for test before actual release branch is generated

Reply:
I see, just noticed the branch name is not the target one.